client/packages/cli_new/src/cli.ts

```diff
@@ -0,0 +1,54 @@
+#!/bin/env node
+import { Command } from "commander"
+import { handleInitCmd } from "./commands/init.ts"
+import { wrapCryptoWaitReady } from "./utils/fns.ts"
+import { handleRegisterCmd } from "./commands/register/register.ts"
+import { handleSubmitCmd } from "./commands/submit/submit.ts"
+import { handleSetOperational } from "./commands/set_operational.ts"
+import { handleBidCmd } from "./commands/bid.ts"
+
+const program = new Command()
+
+program
+  .name("t3rn CLI")
+  .description("CLI for interacting with the t3rn circuit")
+  .version("0.1.0")
+
+program
+  .command("init")
+  .option("-c, --config [file-path]", "Generate a config template")
+  .option("-t, --transfer [file-path]", "Generate a transfer template")
+  .description("Generate a config or transfer template")
+  .action(handleInitCmd)
+
+program
+  .command("register")
+  .option("-g, --gateway <id>", "ID of the gateway to register")
+  .description("Register a gateway with the t3rn circuit")
+  .action(wrapCryptoWaitReady(handleRegisterCmd))
+
+program
+  .command("submit")
```

Comment:
can we rename this to submit-sfx or something? Extrinsic is very broad and refers to any type of on-chain transaction.

Reply:
Hmm, but the same command is used to submit headers.
Usage:
```bash
pnpm cli submit --headers
```

Let's rename the `extrinsic` argument to `sfx`.
Usage:
```bash
# Before
pnpm cli submit --extrinsic transfer.json

#After
pnpm cli submit --sfx transfer.json
```

Reply:
See to commit cdc64d300e9ae36c3e2973b2f5110c73381f3530.